frontend/src/partners/ValidationPartner.js

```diff
@@ -0,0 +1,58 @@
+import * as yup from 'yup'
+import { CUSTOMER_TYPE } from '../Const'
+
+export default function schema() {
+  const schema = yup.object().shape({
+    partnerEmail: yup
+      .string()
+      .required('Kötelező mező!')
+      .email('Nem megfelelő email cím!'),
+    telefon: yup
+      .string()
+      .required('Kötelező mező!')
+      .matches(/([0-9]+)/, 'Nem megfelelő formátum!'),
+    megrendeloTipusa: yup.string().required('Kötelező mező!'),
+    nev: yup.string().required('Kötelező mező!'),
+    rovidNev: yup.string(),
+    adoszam: yup.string().when('megrendeloTipusa', {
+      is: CUSTOMER_TYPE.LEGAL,
+      then: yup
+        .string()
+        .required('Kötelező mező!')
+        .length(8, 'Az adószám 8 számjegyet kell tartalmazzon!')
+        .matches(/([0-9]{8})/),
+      otherwise: yup.string(),
+    }),
+    kadoszamtipus: yup.number().when('megrendeloTipusa', {
+      is: CUSTOMER_TYPE.LEGAL,
+      then: yup
+        .number()
+        .required('Kötelező mező!')
+        .min(1, 'A kadószám 1 és 5 közötti szám!')
```

Comment:
Talán felhasználóbarátabb lenne a kadószám helyett a közösségi adószám.

Reply:
Vagy nem, ez a nem a közösségi adószám, ez az ÁFA típus. Akkor nem tudom miért kadószám a neve, de mindenesetre felhasználóként nem tudnám mi a baj ebből a hibaüzenetből.

Reply:
így kérték, backenden is így várják, de akkor a label-t módosítom ÁFA-kódra.